docs/faq/README.md

```diff
@@ -78,3 +78,19 @@ If either symbol fails to display correctly, your system is still misconfigured.
 Unfortunately, getting font configuration correct is sometimes difficult. Users
 on the Discord may be able to help. If both symbols display correctly, but
 you still don't see them in starship, [file a bug report!](https://github.com/starship/starship/issues/new/choose)
+
+## How do I uninstall Starship?
+
+Starship is just as easy to uninstall as it is to install in the first place.
+
+1. Remove any lines in your shell config (e.g. `~/.bashrc`) used to initialize Starship.
+1. Delete the Starship binary.
+
+If Starship was installed using a package manager, please refer to their docs for uninstallation instructions.
+
+If Starship was installed using the `curl | bash` script, the following command will delete the binary:
+
+```sh
+# Locate and delete the starship binary
+rm -rf $(which starship)
```

Comment:
This is potentially dangerous if the user makes poor file naming choices (e.g. if starship is installed at `/Users/<user>/Documents 2/programs/starship`, this will cause a deletion of their `~/Documents` folder).

```suggestion
rm "$(which starship)"
```

Reply:
Good call! 👍